### ISSUE TITLE
Add a violation report example

### ISSUE THREAD

**Issue by fmarier** [w3c/webappsec#189]
The CSP spec includes an [example violation report](http://www.w3.org/TR/CSP2/#example-violation-report) and since we intend to use the same mechanism in SRI, we should describe what that report looks like (e.g. what fields it is expected to contain).

**Comment by devd** [comment]
What did you implement in Firefox?

**Comment by fmarier** [comment]
I don't have reporting in Firefox yet and I'm planning to land the initial implementation without it.

**Comment by devd** [comment]
aah ok @metromoxie have you implemented reports yet? Might be easier to start with existing implementations.

**Comment by metromoxie** [comment]
No, we do not have reporting yet; just console logging. I have a few other issues (e.g. requiring CORS for SRI) to fix before I can get to reporting.

**Comment by devd** [comment]
tbh, based on my experience, a JS event might be good enough. We could just ask for "sri" field on the onerror event or something like that.

**Comment by fmarier** [comment]
Based on some feedback we have received on CSP, it might be good to specify the report format instead of just providing an example. The small differences in how Chrome and Firefox produce the reports are making life a little harder for people who have to parse/filter these reports.

**Comment by devd** [comment]
aah .. what I am suggesting though is that it is not even clear how critical "reporting" is as opposed to just throwing an error event.

**Comment by metromoxie** [comment]
I'm good either way. We could just go with an error event and then punt reporting until v2, as well.

**Comment by fmarier** [comment]
If it's not going to prevent people from adopting SRI, then I'm ok for moving reporting to v2.
